### PR TITLE
docker: reduce images size

### PR DESCRIPTION
**Pull request by glimchb**
By introducing multi-stage build
And switching to alpine
And using grpcurl from docker

x20 size reduction !

```
$ docker images | grep marvell
opi-marvell-bridge                      main           fe68a9af4097   2 minutes ago    74.2MB
ghcr.io/opiproject/opi-marvell-bridge   main           6211712b57b3   23 hours ago     1.72GB
```

Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Comment by glimchb** [comment]
same as https://github.com/opiproject/opi-nvidia-bridge/pull/44 and https://github.com/opiproject/opi-smbios-bridge/pull/7

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: hadolint/hadolint-action@v3.0.0
         with:
           recursive: true
-          ignore: DL3041
```
Comment (mestery) [inline .github/workflows/linters.yml:29]: Is this change related?

**Review by mestery — APPROVED** [review]

File: .github/workflows/linters.yml (hunk 1 above)
Comment (glimchb) [reply .github/workflows/linters.yml:29]: Yes, since we changed to alpine, need to update ignore rule as well
